in_toto/util.py

```diff
@@ -134,12 +138,44 @@ def prompt_password(prompt="Enter password: "):
   return getpass.getpass(prompt, sys.stderr)
 
 
+def prompt_import_private_key_from_file(filepath, key_type):
+  """Tries to load a key with/without password. If a CryptoError
+  occurs, prompts the user for a password and trys to load the the key again.
+  """
+  try:
+    if key_type == KEY_TYPE_ED25519:
+      key = prompt_import_ed25519_privatekey_from_file(filepath)
+    elif key_type == KEY_TYPE_RSA:
+      key = prompt_import_rsa_key_from_file(filepath)
+    else:  # pragma: no cover
+      # This branch is never possible as argparse already checks valid keys
+      # via the choices parameter.
+      raise TypeError('Unsupported keytype: ' + str(key_type))
+  except:
```

Comment:
We should avoid using a bare except.  Our code style guidelines provides more info on why to avoid them.
https://github.com/secure-systems-lab/code-style-guidelines#use-class-based-exceptions